<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAdminWindow</class>
 <widget class="QWidget" name="CreateAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createAdminPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Створити адміністратора</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>621</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2">
     <widget class="QLineEdit" name="telephoneLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="firstname">
      <property name="text">
       <string>Ім'я</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lastname">
      <property name="text">
       <string>Прізвище</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="middlename">
      <property name="text">
       <string>По батькові</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lastnameLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="telephone">
      <property name="text">
       <string>Номер телефона</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="firstnameLineEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="loginLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="middlenameLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="passport">
      <property name="text">
       <string>Номер паспорта</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLineEdit" name="adressLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="login">
      <property name="text">
       <string>Логін</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="adress">
      <property name="text">
       <string>Адреса проживання</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="passportLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="password">
      <property name="text">
       <string>Пароль</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>621</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Створення адміністратора&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="roleCheckBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Головний адміністратор</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
